fingertip/plugins/cleanup.py

```diff
@@ -22,13 +22,16 @@ def main(what=None, older_than=0):
         return
     if what == 'everything':
         return everything()
+    if what == 'periodical':
```

Comment:
I'm pretty confident it's "periodic".

Reply:
It can be both of them, but `periodic` is shorter and more fitting. But [your first suggestion](https://github.com/t184256/fingertip/pull/10#discussion_r401837002) was `periodical` so I thought you are in favor of this one :)

Reply:
I've reconsidered my ways since then =) Please use 'periodic', sorry for misleading you.

Reply:
ack